Lib/ufo2ft/featureCompiler.py

```diff
@@ -74,7 +74,7 @@ class BaseFeatureCompiler:
     layout tables from these.
     """
 
-    def __init__(self, ufo, ttFont=None, glyphSet=None, **kwargs):
```

Comment:
I can't exactly remember, but maybe that `**kwargs` that you removed was intentional?

Reply:
nevermind, it seems it's ok to remove it.

by the way, the BaseFeatureCompiler.__init__ has a docstring describing each Args, I think you could add an entry for the new extraSubstitutions there